colossalai/pipeline/stage_manager.py

```diff
@@ -174,3 +169,10 @@ def init_process_group_by_stages(self, stages: List[int]) -> ProcessGroup:
             ProcessGroup: Process group of the given stages.
         """
         return self.pg_mesh.get_group_along_axis(self.pipeline_axis, stages)
+
+    @contextlib.contextmanager
+    def set_model_chunk_id(self, model_chunk_id: int):
```

Comment:
`switch_model_chunk_id()` may be clearer

Reply:
3b84204 fix this.